addons/crm/views/crm_lead_views.xml

```diff
@@ -576,47 +576,114 @@
                     <field name="campaign_id" optional="hide"/>
                     <field name="medium_id" optional="hide"/>
                     <field name="source_id" optional="hide"/>
+                    <field name="company_currency" invisible="1"/>
+                    <field name="planned_revenue" sum="Expected Revenues" optional="show" widget="monetary" options="{'currency_field': 'company_currency'}"/>
+                    <field name="stage_id" optional="show" decoration-bf="1"/>
+                    <field name="active" invisible="1"/>
+                    <field name="probability" optional="hide"/>
+                    <field name="company_id" groups="base.group_multi_company" optional="show"/>
                     <field name="tag_ids" optional="hide" widget="many2many_tags" options="{'color_field': 'color'}"/>
-                    <field name="priority" optional="hide"/>
-                    <field name="activity_exception_decoration" widget="activity_exception"/>
+                    <field name="referred" invisible="1"/>
+                    <field name="message_needaction" invisible="1"/>
                 </tree>
             </field>
         </record>
 
+        <record id="crm_lead_view_list_activities" model="ir.ui.view">
+            <field name="name">crm.lead.list.activities</field>
+            <field name="model">crm.lead</field>
+            <field name="mode">primary</field>
+            <field name="priority">15</field>
+            <field name="inherit_id" ref="crm.crm_case_tree_view_oppor"/>
+            <field name="arch" type="xml">
+                <field name="user_id" position="attributes">
+                    <attribute name="optional">hide</attribute>
+                </field>
+                <field name="team_id" position="attributes">
+                    <attribute name="optional">hide</attribute>
+                </field>
+                <field name="message_needaction" position="after">
+                    <button name="action_snooze" type="object" string="Snooze 7d" icon="fa-bell-slash" />
+                    <button name="%(crm.crm_lead_act_window_compose)d" type="action" string="Email" icon="fa-envelope"
+                            context="{
+                                        'default_composition_mode': 'comment',
+                                        'default_use_template': True,
+                                    }"
+                    />
+                </field>
+            </field>
+        </record>
+
+        <record id="view_crm_case_my_activities_filter" model="ir.ui.view">
+            <field name="name">crm.lead.search.myactivities</field>
+            <field name="model">crm.lead</field>
+            <field name="inherit_id" ref="crm.view_crm_case_leads_filter"/>
+            <field name="arch" type="xml">
+                <xpath expr="//filter[@name='activities_overdue']" position="replace">
```

Comment:
Can't you do something like
```
<xpath expr="//filter[@name='activities_overdue']" position="attributes">
    <attribute name="invisible">0</attribute>
</xpath>
```
This should allow to avoir redefining the whole filter.

Reply:
I tried, but it wasn't working... did I miss something?

Reply:
Well I am quite sure it worked, for example in mass_mailing_sms we use it to hide mail-related fields of mailing. What was the issue ? Not displaying, or view error ?

Reply:
Up this one, if it does not work could you check with JS team maybe ?

Reply:
I tried exactly that way but it still doesn't work. LNA tried also and got the same result.

I asked AAB and he told me his team has no time for the moment and it's nothing related to js. I will look at it more carefully.

Reply:
Ok. Then keep it like it currently, just add a comment above the redefinition telling why we do it like it and that this is temporary. That way we can move forward with this task and look at the underlying bug in master without being in a rush.